perfeed/tools/weekly_summarizer.py

```diff
@@ -0,0 +1,98 @@
+import asyncio
+import time
+from datetime import datetime, timedelta
+
+from jinja2 import Environment, StrictUndefined
+
+from perfeed.config_loader import settings
+from perfeed.git_providers.base import BaseGitProvider
+from perfeed.git_providers.github import GithubProvider
+from perfeed.llms.base_client import BaseClient
+from perfeed.llms.ollama_client import OllamaClient
+from perfeed.models.pr_summary import PRSummary
+from perfeed.tools.pr_summarizer import PRSummarizer
+from perfeed.utils.utils import json_output_curator
+
+
+# python perfeed.py -users jimmytai chihangwang -repo perfeed-backend -period 1w
```

Comment:
Do we want to make the owner as one of the input?

Reply:
Yes, good catch